src/rules/require-top-level-describe.ts

```diff
@@ -0,0 +1,75 @@
+import { Rule } from 'eslint';
+import { getAmountData } from '../utils/misc';
+import * as ESTree from 'estree';
+import { isDescribeCall, isTest, isTestHook } from '../utils/ast';
+
+export default {
+  create(context) {
+    const { maxTopLevelDescribes } = {
+      maxTopLevelDescribes: Infinity,
+      ...((context.options?.[0] as {}) ?? {}),
+    };
+
+    let topLevelDescribeCount = 0;
+    let describeCount = 0;
+
+    return {
+      CallExpression(node) {
+        if (isDescribeCall(node)) {
+          describeCount++;
+
+          if (describeCount === 1) {
+            topLevelDescribeCount++;
+
+            if (topLevelDescribeCount > maxTopLevelDescribes) {
+              context.report({
+                node,
+                messageId: 'tooManyDescribes',
+                data: getAmountData(maxTopLevelDescribes),
+              });
+            }
+          }
+        } else if (!describeCount) {
+          if (isTest(node)) {
+            context.report({ node, messageId: 'unexpectedTest' });
+          } else if (isTestHook(node)) {
+            context.report({ node, messageId: 'unexpectedHook' });
+          }
+        }
+      },
+      'CallExpression:exit'(node: ESTree.CallExpression) {
+        if (isDescribeCall(node)) {
+          describeCount--;
+        }
+      },
+    };
+  },
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description:
+        'Require test cases and hooks to be inside a `test.describe` block',
+      recommended: true,
```

Comment:
Looks recommended?

Reply:
Ah good catch.  Updated this to false and fixed a few other rules which were marked incorrectly